File: cpp/bmi.cpp
/*
 * bmi.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    float a = 0;
    float b = 0;
    float c = 0;
    int ile = 0;
    int i = 0;
    
    cout << "Ile razy chcesz skorzystac z programu?" << endl;
    cin >> ile;
    
    while (i < ile)
    {
        cout << "Podaj masę ciała:" << endl;
        cin >> a;
        cout << "Podaj wzrost:" << endl;
        cin >> b;
        
        while (b > 4)
        {
            b = b/10;
        }
        
        c=a/(b*b);
        cout << c << endl;
        
        if (c < 18.5)
        {
            cout << "niedowaga" << endl;
        }
        else if (c < 25 )
        { 
            cout << "norma" << endl;
        }
        else if (c < 30)
        { 
            cout << "nadwaga" << endl;
        }
        else if (c >= 30)
        { 
            cout << "otyłość" << endl;
        }
        i = i + 1;
    }
    return 0;
}

File: python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    ile = int(input("Ile razy chcesz skorzystac z programu?"))
    i = 0

    while i < ile:
        a = float(input('Podaj masę ciała: '))
        b = float(input('Podaj wzrost: '))

        while b > 3:
            b = b / 10

        c = float(a / (b * b))

        print(c)

        if c < 18.5:
            print ("niedowaga")
        elif c < 25:
            print ("norma")
        elif c < 30:
            print ("nadwaga")
        elif c >= 30:
            print ("otyłość")
        i = i + 1

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
